clang/test/Driver/flang/emit-flang-attrs.f90

```diff
@@ -0,0 +1,57 @@
+! REQUIRES: aarch64-registered-target
```

Comment:
Does this test `REQUIRES: classic-flang`? Or will it work with LLVM Flang?

Reply:
> Can the commits [222b18f](https://github.com/flang-compiler/classic-flang-llvm-project/commit/222b18ff15c1445ef07cc79551e90d83d1225933), [84262af](https://github.com/flang-compiler/classic-flang-llvm-project/commit/84262af9e398518d0792c9e213ffe5295c762286), [5a9a222](https://github.com/flang-compiler/classic-flang-llvm-project/commit/5a9a2229f8274dcc780ecf73a1be0af3ac63a424) be upstreamed in a form that works with both Classic Flang and LLVM Flang?

No, at least, not by us.